src/DotLiquid.Tests/ConditionTests.cs

```diff
@@ -223,10 +223,12 @@ public void TestCapitalInCustomOperator()
             try
             {
                 Condition.Operators["IsMultipleOf"] =
-                    (left, right) => (int)left % (int)right == 0;
+                    (left, right) => (long)left % (long)right == 0;

```

Comment:
We need to preserve original support for int while allowing new custom operator to handle long values when required by the value exceeding the int range.